src/sdl2_compat.c

```diff
@@ -1333,6 +1334,31 @@ SDL_LOG_IMPL(Error, ERROR)
 SDL_LOG_IMPL(Critical, CRITICAL)
 #undef SDL_LOG_IMPL
 
+static void SDLCALL LogOutputFunction3to2(void *userdata, int category, SDL_LogPriority priority, const char *message)
+{
+    LogOutputFunction2(userdata, category, LogPriority3to2(priority), message);
+}
+
+SDL_DECLSPEC void SDLCALL SDL_LogSetOutputFunction(SDL2_LogOutputFunction callback, void *userdata)
+{
+    if ((SDL_LogOutputFunction)callback == SDL3_GetDefaultLogOutputFunction()) {
```

Comment:
Hmm, looks like you can pass a NULL callback too to disable log output, I better handle that as well...